lib/log/event.rb

```diff
@@ -0,0 +1,16 @@
+require 'json'
+
+module Log
+  class Event
+
+    def initialize(name, monitored=false, context={})
```

Comment:
The API could be better here. If `monitored` is a keyword argument, calls to `Log::Event.new` are much more readable:

```
Log::Event.new('schema_violation', true, { ...})
```
vs
```
Log::Event.new('schema_violation', { ... }, monitored: true)
```

It's clear what's going on, `true` alone does not tell anything.

Further read: https://robots.thoughtbot.com/ruby-2-keyword-arguments (Keyword arguments vs positional arguments)